build.sh

```diff
@@ -89,7 +89,7 @@ publish()
     global_packages=$(dotnet nuget locals -l global-packages)
     global_packages=${global_packages/global-packages: /""}
     run_command rm "-rf" "$global_packages/zeroc.slice/$version" "$global_packages/icerpc/$version" "$global_packages"/icerpc.*/"$version"
-    run_command dotnet "nuget" "push" "tools/**/$dotnet_config/*.$version.nupkg" "--source" "$global_packages"
+    run_command dotnet "nuget" "push" "tools/**/$dotnet_config/*.$version*.nupkg" "--source" "$global_packages"
```

Comment:
Is `*` missing from the line below, in case we update some src packages, like for example IceRpc.Templates?

Reply:
Windows script doesn't handle this versions well, need to fix it.